ui/src/app/shared/service/local-storage.service.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable, inject} from '@angular/core';
```

Comment:
I am guessing that the built-in local storage in Angular has been removed and we instead roll our own?

Reply:
`'angular-2-local-storage'` isn't exactly a native Angular lib... it probably isn't supported anymore hence our own service for local storage